backend/schema/corpora_schema.md

```diff
@@ -296,8 +205,8 @@ X_umap for example.
 
 Finally, the dataset-level metadata is stored in `uns`, which is just key-value pairs.
 
-Note that anndata supports "layers" and "raw" values for counts. Those are permitted, but Corpora will treat `X` as the "final" matrix for further analysis
-and visualization. Once anndata [unifies its treatment of layers](https://github.com/theislab/anndata/issues/244), Corpora will use the "default" as the
+Note that anndata supports "layers" and "raw" values for counts. Those are permitted, but cellxgene will treat `X` as the "final" matrix for further analysis
+and visualization. Once anndata [unifies its treatment of layers](https://github.com/theislab/anndata/issues/244), cellxgene will use the "default" as the
 final matrix, however that ends up being specified. In anndata files, the layer_descriptions dictionary should have a key "X" and optionally "raw.X" to
 describe those layers.
```

Comment:
Need clarification. My understanding is that the "raw" matrix is required. So, in the layer_descriptions dictionary, the "raw.X" is optional? What if there is no optional "final" matrix layer? Does the layer_descriptions dictionary still need to have key "X"?

Reply:
There are unfortunately two meanings of "raw". One is a description of values in a matrix: it basically means counts that haven't been normalized, etc. The other is an attribute in an AnnData object, accessed like `adata.raw`

There must be one layer in an AnnData that is raw (first meaning). It doesn't particularly matter where that is, but a natural place for it is in the `.raw` attribute.